PageFlip/src/main/java/com/eschao/android/widget/pageflip/PageFlip.java

```diff
@@ -598,13 +600,13 @@ private void createPages() {
         if (mPageMode == AUTO_PAGE_MODE &&
             mViewRect.surfaceW > mViewRect.surfaceH) {
             mPages[FIRST_PAGE] = new Page(mViewRect.left + marginLeft, 0,
-                                          mViewRect.top + marginTop, mViewRect.bottom - marginBottom);
+                                          mViewRect.top - marginTop, mViewRect.bottom + marginBottom);
             mPages[SECOND_PAGE] = new Page(0, mViewRect.right - marginRight,
```

Comment:
Why the first and the second page does not have the same size ?

Reply:
Ah it's because it's not size but position, right?

Reply:
exactly